packages/base-zone/src/prepare-revocable.js

```diff
@@ -87,6 +95,8 @@ export const prepareRevocableKit = (
 
   const revocableKindName = `${uKindName}_caretaker`;
 
+  let amplifier;
```

Comment:
This default of `amplifier: any` leaves behind some typing weeds that may later grow larger.
```suggestion
  /** @type {(revocable: U) => RevocableKit<U> | undefined */
  let amplifier;
```

Reply:
Trying `Amplify<RevocableKit<U>> | undefined` instead

Reply:
Backed out.